config/locals.json

```diff
@@ -3,7 +3,7 @@
     "title": "Etcher",
     "prefix": "",
     "slogan": "Burn. Better.",
-    "lead": "Burn images to SD cards & USB drives, safe & easy.",
+    "lead": "Burn images to SD cards & USB drives, safely and easily.",
```

Comment:
Ahh, sorry I forgot to commit the changes to this file too, but that's because I was using the GH web-editor rather than building locally :-/

To prevent similar slip-ups in future, I guess you could run `git status -s` after running `yarn run export` and checking that git-status returns an empty string (i.e. none of the files in the repo got modified). See e.g. https://github.com/resin-io/etcher/blob/master/scripts/ci/ensure-staged-shrinkwrap.sh